src/mpyl/steps/deploy/k8s/__init__.py

```diff
@@ -53,6 +89,54 @@ def upsert_namespace(logger: Logger, step_input: Input, context: str):
     return namespace
 
 
+def get_config_map(context: str, namespace: str, config_map_name: str) -> V1ConfigMap:
+    config.load_kube_config(context=context)
+    api = client.CoreV1Api()
```

Comment:
I think I would pass the `api`, reduces code duplication (with the functions below) and ensures the config/api is only loaded/instantiated once